Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:22.04
```

Comment:
maybe it's time to move on to 24.04..? (not necessarily in this PR)

Reply:
This file is a legacy; it is from artifact generating script, which is not part of the benchmark. Therefore, I have removed it.

Reply:
I have just tested the benchmark with 24.04 and found that it works well. Therefore, I have upgraded it.